TaskWithMaxMinSortForArray/TaskWithMaxMinSortForArray.cpp

```diff
@@ -0,0 +1,59 @@
+﻿#include <iostream>
+
+int searchMax(int array[], size_t len) {
```

Comment:
Can you explain why one of the arguments has a type int[]? If it's a hint that your array is in stack, then that's not true you know.

Reply:
This argument is passed as an int[] array of integer values, and in the function we refer to the elements of this array, without these [] does not work

Reply:
Fixed